src/tests/ftest/control/dmg_pool_query_ranks.py

```diff
@@ -23,7 +23,7 @@ def setUp(self):
         super().setUp()
 
         # Init the pool
-        self.add_pool(connect=False)
+        self.add_pool(connect=False, properties='rd_fac:1')
```

Comment:
We should instead put the properties in the config `dmg_pool_query_ranks.yaml`. Under the existing `pool` key:
```
pool:
  properties: rd_fac:1
```

Reply:
Fixed.